src/test/resources/dokumentasjon/features/mapping/pdl/kontaktadresser.feature

```diff
@@ -0,0 +1,84 @@
+# language: no
+# encoding: UTF-8
+
+Egenskap: Mapping av kontaktadresser fra PDL HentPerson.Kontaktadresse
+
+  TODO:
+  * Er dette realistiske eksempler?
+
+  Scenario: En person som har flyttet flere ganger i Norge
+    Gitt følgende kontaktadresser fra PDL:
+      | Utenlandsk adresse landkode | Gyldig fra og med   | Gyldig til og med   |
+      |                             |                     | 2015-03-25 10:03:03 |
+      |                             | 2016-04-25 14:03:03 |                     |
+
+    Når kontaktadresser mappes
+
+    Så skal mappede kontaktadresser være
+      | Landkode | Fra og med dato | Til og med dato |
+      | NOR      |                 | 2015-03-25      |
+      | NOR      | 2016-04-25      |                 |
+
```

Comment:
Jeg tenker her at hvis en bruker har noe på utenlandskadresser f.eks en dato så vil aldri landkoden bli NOR men ukjent. Hvis denne testen gir grønt, bør dette endres på. Hvis hele objektet er null, da skal landkoden bli NOR for da er det registrert data på et annet felt som f.eks vegadresse.

Reply:
Ah, glem det gyldig fra og med ligger vell i roten av kontaktadresser, jeg misforstod.

Reply:
Men for lesbarhet skyld, så kanskje denne burde vært omskrevet til å hatt 

"Gitt følgende datoer på kontaktadresser fra PDL"

Med underfelt: 

"Gitt utenlandsk adresse inneholder"

Men hvis Helle skjønner det, så er det fint for min del:)